packages/teams-js/src/public/sharing.ts

```diff
@@ -194,4 +198,68 @@ export namespace sharing {
   export function isSupported(): boolean {
     return ensureInitialized(runtime) && runtime.supports.sharing ? true : false;
   }
+
+  /**
+   * Namespace to get the list of content shared in a context
+   *
+   * @beta
+   */
+  export namespace history {
+    /**
+     * Represents the data returned when calling {@link sharing.history.getContent}
+     *
+     * @beta
+     */
+    export interface IContentResponse {
+      /** Id of the app where the content was shared from */
+      appId: string;
+      /** Title of the shared content */
+      title: string;
+      /** Reference of the shared content */
+      contentReference: string;
+      /** Id of the thread where the content was shared. This is a UUID */
+      threadId: string;
+      /** Id of the user who shared the content. This is a UUID */
+      author: string;
+      /** Type of the shared content.
+       * For sharing to Teams stage scenarios, this value would be ShareToStage
```

Comment:
>ShareToStage [](http://example.com/codeflow?start=67&length=12)

Please surround this with backticks, e.g.,

For sharing to Teams stage scenarios, this value would be `ShareToStage`

And also add:
Other `contentType` values will be added here over time
(or some other text that indicates that there are currently no other known values but it will expand in the future)

Reply:
Updated and added the comments.